CHANGES.md

```diff
@@ -71,11 +71,14 @@
     More details will be in an upcoming [blog post](https://beam.apache.org/blog/python-improved-annotations/index.html).
 * Improved the Interactive Beam API where recording streaming jobs now start a long running background recording job. Running ib.show() or ib.collect() samples from the recording ([BEAM-10603](https://issues.apache.org/jira/browse/BEAM-10603)).
 * In Interactive Beam, ib.show() and ib.collect() now have "n" and "duration" as parameters. These mean read only up to "n" elements and up to "duration" seconds of data read from the recording ([BEAM-10603](https://issues.apache.org/jira/browse/BEAM-10603)).
+* Initial preview of [Dataframes](https://s.apache.org/simpler-python-pipelines-2020#slide=id.g905ac9257b_1_21) support.
+    See also example at apache_beam/examples/wordcount_dataframe.py
 * X feature added (Java/Python) ([BEAM-X](https://issues.apache.org/jira/browse/BEAM-X)).
 
 ## Breaking Changes
 
-* X behavior was changed ([BEAM-X](https://issues.apache.org/jira/browse/BEAM-X)).
+* Python 2 and Python 3.5 support dropped.
+* Pandas 1.x allowed.  Older version of Pandas may still be used, but may not be as well tested.
```

Comment:
Should this comment be part of the Dataframes note above? I don't think pandas 1.x support has any broader implications

Reply:
I think it makes more sense here--the main implication here is how diamond dependencies might get resolved.

Reply:
Gotcha, makes sense